reference/library/src/morpheus-master/sass/modules/tool/roster/_roster.scss.bak

```diff
@@ -0,0 +1,242 @@
+##{$namespace}sakai-roster {
```

Comment:
Of course. It is essential. Not really - I was in a rush. Quite the refactor this was.